.github/workflows/phpunit-mysql.yml

```diff
@@ -45,14 +56,11 @@ jobs:
   phpunit-mysql:
     runs-on: ubuntu-latest
 
-    needs: changes
+    needs: [changes, matrix]
     if: needs.changes.outputs.src != 'false'
 
     strategy:
-      matrix:
-        php-versions: ['8.0', '8.1', '8.2', '8.3']
-        server-versions: ['master']
-        mysql-versions: ['8.1']
+      matrix: ${{ fromJson(needs.matrix.outputs.matrix) }}
```

Comment:
I'm not a huge fan of this at the moment, as it could easily break without someone noticing 🙈